app/templates/services/_base_service_page.html

```diff
@@ -78,30 +75,27 @@
           ],
           field_headings_visible=False
         ) %}
-          {% call summary.row(complete=not question.answer_required) %}
+          {% call summary.row() %}
             {{ summary.field_name(question.label) }}
-            {% if question.answer_required %}
-              {% call summary.field() %}
-                {% if framework and framework.status == 'open' %}
-                    <a href="{{ url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, _anchor=question.id) }}">Answer required</a>
+            {% if question.is_empty %}
+              {% if framework and framework.status == 'open' and section.edit_questions %}
+                {% if question.is_empty %}
+                  {{ summary.link('Answer question', url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, question_slug=question.slug)) }}
                 {% else %}
-                  Not answered
+                  {{ summary.field("Not answered") }}
                 {% endif %}
-              {% endcall %}
-            {% elif question.is_empty %}
-              {% call summary.field() %}
-                <span class="summary-item-field-answer-required">{{ question.empty_message }}</span>
-              {% endcall %}
+              {% endif %}
             {% else %}
               {{ summary[question.type](question.value, question.assurance) }}
             {% endif %}
-            {% if section.edit_questions and framework.status == 'open' %}
-              {% if not question.is_empty %}
-                {{ summary.remove_link('Remove', url_for(".remove_subsection", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, question_slug=question.slug)) }}
+            {% if framework.status == 'open' %}
+              {% if not question.is_empty and section.edit_questions %}
+                {{ summary.edit_link('Edit', url_for(".edit_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id, section_id=section.id, question_slug=question.slug)) }}
               {% else %}
-                {{ summary.text("") }}
+                {% call summary.field(action=True) %}
+                  {{ 'Optional' if question.get('optional') else '' }}
```

Comment:
This is another thing that looks odd for DOS... can @ralph-hawkins live with this?

![screen shot 2017-02-10 at 12 17 58](https://cloud.githubusercontent.com/assets/6525554/22826555/017b0296-ef8b-11e6-9efd-b49574d0d0a1.png)